dts/bindings/mfd/x-powers,axp2101.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2023, Martin Kiepfer <mrmarteng@teleschirm.org>
+# Copyright (c) 2024 Deng Baoan <dengbaoan@foxmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: X-Powers AXP2101
+
+compatible: "x-powers,axp2101"
+
+include: i2c-device.yaml
+
+properties:
+  reg:
+    required: true
+
+  vbusen-disable:
```

Comment:
Add `drivers/mfd/mdf_axp2101.c` to handle this property.

I think this property should be good to change as `vbus-enable`.